website/pages/en/index.js

```diff
@@ -0,0 +1,102 @@
+const React = require('react')
```

Comment:
Tried using `import ... from` syntax but then the doc server refuses to start :(

Reply:
🤷‍♂️